packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -194,9 +194,12 @@ const emit = defineEmits<{
       line-height: 1.5rem;
     }
 
+    &--loading &__label {
+      visibility: hidden;
+    }
+
     &__loading {
-      width: var(--onyx-spacing-3xl);
-      height: 100%;
+      position: absolute;
```

Comment:
Shoudln't we also make the parent position relative?

Reply:
done